honeybadgermpc/robust_reconstruction.py

```diff
@@ -13,17 +13,17 @@
     return done, pending
 
 
-def attempt_reconstruct(encoded, field, n, t, point):
+def attempt_reconstruct(encoded, field, n, t, point, precomputed_data=None):
     # Attempt to reconstruct with a mixture of erasures or errors
     assert len(encoded) == n
     assert sum(f is not None for f in encoded) >= 2*t + 1
 
     # raise ValueError("Sentinel bug")
 
     # interpolate with error correction to get f(j,y)
-    _, decode, _ = make_encoder_decoder(n, t+1, field.modulus)
+    _, decode, _ = make_encoder_decoder(n, t+1, field.modulus, point)
 
-    p = polynomials_over(field)(decode(encoded))
+    p = polynomials_over(field)(decode(encoded, precomputed_data=precomputed_data))
```

Comment:
nit: consider renaming the arg value and name

Reply:
Are you referring to precomputed_data?

Reply:
I think this is fine for function args. Closing this.